src/styles/index.scss

```diff
@@ -1,3 +1,250 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
The body element is using flexbox with 'align-items: center', which centers the content horizontally. Ensure this is the intended behavior, as it might affect the layout of the page if not handled correctly.